doc/Known_issues/Troubleshooting.md

```diff
@@ -0,0 +1,29 @@
+# Troubleshooting
+
+Please follow steps below if you encounter issues when using Windows Contrail.
+
+## 1. Diagnostic check
+
+Run `Invoke-DiagnosticCheck.ps1` script from [tools repository](https://github.com/Juniper/contrail-windows-tools)
+on Windows Computes that cause problems. Refer to README in root of the repository
+for script configuration info.
+
+**Note**: the script can be ran with or without Administrator privileges. However, some checks
+will not be performed without them.
+
+## 2. Known issues
+
+Search [known issues section](./Known_issues.md) for symptoms of your issue.
+The section contains recovery procedures.
+
+## 3. Total cleanup
+
+Run `Clear-ComputeNode.ps1` script from [tools repository](https://github.com/Juniper/contrail-windows-tools)
```

Comment:
Maybe we can mention here that there is a tool to invoke Clear-ComputeNode.ps1 on many computes remotely.